pkg/services/ngalert/api/forked_am.go

```diff
@@ -0,0 +1,183 @@
+package api
+
+import (
+	"fmt"
+
+	apimodels "github.com/grafana/alerting-api/pkg/api"
+	"github.com/grafana/grafana/pkg/api/response"
+	"github.com/grafana/grafana/pkg/models"
+	"github.com/grafana/grafana/pkg/services/datasources"
+)
+
+type ForkedAMSvc struct {
+	AmSvc, GrafanaSvc AlertmanagerApiService
+	DatasourceCache   datasources.CacheService
+}
+
+func NewForkedAM(datasourceCache datasources.CacheService, proxy, grafana AlertmanagerApiService) *ForkedAMSvc {
+	return &ForkedAMSvc{
+		AmSvc:           proxy,
```

Comment:
Shouldn't this be named `AMSvc`, "AM" referring to AlertManager? Would also help since it's consistent with `ForkedAMSvc`.
```suggestion
		AMSvc:           proxy,
```